quantum/rgb_matrix/animations/breathing_anim.h

```diff
@@ -2,18 +2,14 @@
 RGB_MATRIX_EFFECT(BREATHING)
 #    ifdef RGB_MATRIX_CUSTOM_EFFECT_IMPLS
 
-bool BREATHING(effect_params_t* params) {
-    RGB_MATRIX_USE_LIMITS(led_min, led_max);
+hsv_t BREATHING_math(hsv_t hsv, uint8_t i, uint8_t time) {
+    time  = scale16by8(g_rgb_timer, rgb_matrix_config.speed / 8);
```

Comment:
Not a huge fan of calling `scale16by8` twice, but the impact should be small, I think. Short of rewriting the runner itself,

Reply:
A simpler runner can be added without a time scale for effects that uses its own. Should this (and the other runner PR changes) wait for it?

Reply:
That would probably be best. But I would say get these in first and then add/update the runners later.

Reply:
Thanks, I can work on that in the next cycle.

Reply:
> Not a huge fan of calling `scale16by8` twice, but the impact should be small, I think. Short of rewriting the runner itself,

Using `time / 2` from runner should scale it down accordingly to the original.